src/python/WMCore/ResourceControl/MySQL/ListThresholdsForCreate.py

```diff
@@ -71,11 +71,7 @@ class ListThresholdsForCreate(DBFormatter):
                             WHERE wmbs_job.location IS NULL AND
                                   wmbs_job_state.name != 'killed' AND
                                   wmbs_job_state.name != 'cleanout' AND
-                                  (wsv.valid = 1 OR
-                                   (wsv.valid IS NULL AND NOT EXISTS
```

Comment:
I had a look at vocms0254 db and I cannot find anything for `wsv.valid IS NULL AND...`.
Still, running the original sub-query (unassigned_jobs) and this one, I get around 246k rows while the latter gives me 245k rows.

Reply:
you can find  wsv.valid is NULL from the table but only when you do the left outer join. Were you checking that? I am still trying to understand original query

Reply:
I'm not sure I got the correct query. However, looking at the code, I see we do not have any update action for wmbs_subscription_validation. So either a subscription has a site white/black list or it doesnt:
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/WorkQueue/WMBSHelper.py#L294

That means, removing this part of the code _should_ indeed be fine